lib/src/lib_edit_address/ui_components/edit_address_form.dart

```diff
@@ -116,7 +114,7 @@ class EditAddressForm<T extends PickerItemModel> extends StatelessWidget {
                 configuration: const TextFieldConfiguration(
                   haveOnlyOneSheet: false,
                 ),
-                dialogHasBottomPadding: false,
+                dialogHasBottomPadding: true,
```

Comment:
This will broke the storybook example. Alternative way to make it work is to expose the property, so we can easily fix the project.

Reply:
I have exposed the property for now from this file [edit_address_form.dart]

Reply:
As we decided the storybook won't be supported anymore

Reply:
I have removed hardcoded ` dialogHasBottomPadding: true,` in both `TextFieldDialog`